tests/substrate_tests/contracts.rs

```diff
@@ -300,7 +300,7 @@ fn mangle_overloaded_function_names_in_abi() {
         .collect();
 
     assert!(messages_a.contains(&"foo".to_string()));
-    assert!(!messages_a.contains(&"foo_".to_string()));
+    assert!(!messages_a.contains(&"foo_bool".to_string()));
```

Comment:
Why did you change this?

Reply:
Forgot to hard reset the main branch, but I figured I can as well smuggle this in :mag: